package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -157,7 +157,7 @@ function M.find_phy(config)
 	end
 end
 
-local function get_addresses(uci, radio)
+local function get_addresses(_, radio)
```

Comment:
As this is a local function, we can also just remove the first argument and adjust the call sites.

Reply:
The `_` "moved up" to `function M.get_wlan_mac(_, radio, index, vif)`